using-kubecost/navigating-the-kubecost-ui/cloud-costs-explorer.md

```diff
@@ -8,46 +8,6 @@ The Cloud Cost Explorer is a dashboard which provides visualization and filterin
 If you haven't performed a successful billing integration with a cloud service provider, the Cloud Cost Explorer won't have cost data to display. Before using the Cloud Cost Explorer, make sure to read our [Cloud Billing Integrations](/install-and-configure/install/cloud-integration/README.md) guide to get started, then see our [specific articles](/install-and-configure/install/cloud-integration/README.md#adding-a-cloud-integration) for the cloud service providers you want to integrate with.
 {% endhint %}
 
-## Installation and configuration
-
-{% hint style="info" %}
-As of v1.104, Cloud Cost is enabled by default. If you are using v1.04+, you can skip the Installation and Configuration section.
-{% endhint %}
-
-For versions of Kubecost up to v1.103, Cloud Cost needs to be enabled first through Helm, using [the following parameters](https://github.com/kubecost/cost-analyzer-helm-chart/blob/a9198777ecd6d1f68f38afb7e42d7cc13e17a1f8/cost-analyzer/values.yaml#L457-L463):
-
-```yaml
-kubecostModel:
-  cloudCost:
-     enabled: true
-     labelList:
-       IsIncludeList: false
-       # format labels as comma separated string (ex. "label1,label2,label3")
-       labels: ""
-     topNItems: 1000
-```
-
-Enabling Cloud Cost is required. Optional parameters include:
-
-* `labelList.labels`: Comma-separated list of labels; empty string indicates that the list is disabled
-* `labelList.IsIncludeList`: If true, label list is a white list; if false, it is a black list
-* `topNItems`: number of sampled "top items" to collect per day
```

Comment:
This doesn't have to be addressed in this PR, but for a future PR.

I agree that we should delete this section since these are now on by default. However it would be good to still find a place to document `labels`, `IsIncludeList`, and `topNItems` since those are still configurable via the Helm chart. https://github.com/kubecost/cost-analyzer-helm-chart/blob/7cb64ae627778d9eac917ad039a0507bbc81a375/cost-analyzer/templates/_helpers.tpl#L1064-L1077

Reply:
There is a thread on Slack about this, apparently these actually don't do anything as of v2.0